merlin_standard_lib/schema/schema.py

```diff
@@ -216,6 +217,18 @@ class Schema(_Schema):
     """A collection of column schemas for a dataset."""
 
     feature: List["ColumnSchema"] = betterproto.message_field(1)
+    _is_first_init = True
+
+    def __post_init__(self):
+        super().__post_init__()
+        if self._is_first_init:
+            # TODO: Make description more descriptive. What version are we planning to remove it?
```

Comment:
@nv-alaiacano @oliverholworthy How should we write this warning? Do we want to commit to a specific release to actually remove it?